src/KeyboardButton.js

```diff
@@ -20,7 +20,7 @@ export default class KeyboardButton extends PureComponent {
 		return (
 			<button
 				type="button"
-				className={`keyboard-button ${this.props.classes}`}
+				className={`keyboard-button ${this.props.classes ? this.props.classes : ''}`}
```

Comment:
@WiaczeslawP agreed, this is mostly just a fix, feel free to edit the PR to your liking. I normally use the`classnames` package but I think that's over kill in this case.

Could amend to 
```
let classes = this.props.classes || '';
render...
```